person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
+    super
```

Comment:
- [ ] The same happens here, please consider adding the parenthesis! 🤓 🔧 

```suggestion
    super()
```